lib/l10n/app_en.arb

```diff
@@ -5,5 +5,6 @@
   "login_validation_error": "Email or password format is not valid. Please try again!",
   "start_survey": "Start Survey",
   "submit_survey": "Submit Survey",
-  "today": "Today"
+  "today": "Today",
+  "answer_text_area_hint": "Your thoughts"
```

Comment:
Hmmmm, I was expecting the API would return the hint (instead of hardcoding it)

Would you be able to double-check? You might want to rebase from [this PR,](https://github.com/nimblehq/ic-flutter-taher-toby/pull/80) in order to get the answers 🙏

Reply:
Conclusion: API is returning `null` values for `text`, `help_text`, and `input_mask_placeholder`.

It's quite unclear, so let's keep it hardcoded